public/locales/en/widgets/location.json

```diff
@@ -0,0 +1,34 @@
+{
+  "form": {
+    "field": {
+      "query": "City / postal code",
+      "latitude": "Latitude",
+      "longitude": "Longitude"
+    },
+    "button": {
+      "search": {
+        "label": "Search",
+        "disabledTooltip": "Please choose a city / postal code first"
+      }
+    },
+    "empty": "Unknown location"
+  },
+  "modal": {
+    "title": "Choose a location",
+    "table": {
+      "header": {
+        "city": "City",
+        "country": "Country",
+        "coordinates": "Coordinates",
+        "population": "Population"
+      },
+      "action": {
+        "select": "Select {{city}}, {{countryCode}}"
+      },
+      "population": {
+        "fallback": "Unknown",
+        "count": "{{count}} people"
+      }
+    }
+  }
+}
```

Comment:
Dont't forget to delete the old translation file

Reply:
There is no old translation file. The `weather.json` is still required and the label in there is also used as title of the card.